presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -86,6 +87,7 @@ public void testReadFullAcidPartitionedBucketed()
     }
 
     @Test(groups = HIVE_TRANSACTIONAL, timeOut = TEST_TIMEOUT)
+    @Flaky(issue = "https://github.com/prestosql/presto/issues/4857")
```

Comment:
FYI A different test method is tagged than originally reported n https://github.com/prestosql/presto/issues/4927

will follow-up